### PR TITLE
feat: Mention that adding people sends emails DOCS-498

### PR DESCRIPTION
**Pull request by nicklem**
We should mention that adding people to an org on Codacy sends them a simple invitation email.

Fixes https://github.com/codacy/docs/issues/1557.

### :eyes: Live preview
https://feat-invitation-email-docs-498--docs-codacy.netlify.app/organizations/managing-people/#adding-people

### CODE REVIEW

File: docs/organizations/managing-people.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ To add members to your organization:
 
     ![Adding members to your organization](images/organization-people-add-modal.png)
 
-1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**.
+1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails an invitation to newly added members.
```
Comment (nicklem) [inline docs/organizations/managing-people.md:44]: Plain and a bit dull, but that's all we need to tell them, really.

Here's my conclusion after some investigation: https://codacy.atlassian.net/browse/DOCS-498?focusedCommentId=53390

Comment (prcr) [reply docs/organizations/managing-people.md:44]: Maybe we can reference that the invitation email includes a link to allow the recipients to log in to Codacy:

> Codacy emails the newly added members inviting them to log in [to Codacy].

File: docs/organizations/managing-people.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ To list and manage the members of your organization, open your organization **Se
 
 To become a member of an organization on Codacy you must [sign up to Codacy](../getting-started/codacy-quickstart.md) using your Git provider and follow the instructions to either join an existing organization or add a new one.
 
-To join or add an organization after having complete the signup process, click **Organizations** on the top right-hand menu under your avatar:
+To join or add an organization after completing the signup process, click **Organizations** on the top right-hand menu under your avatar:
```
Comment (prcr) [inline docs/organizations/managing-people.md:18]: I would never have thought that we would hit this Vale error so fast [after having removed the word `signup` from the accept list](https://github.com/codacy/pulse-user-docs/pull/167#discussion_r1062613135)! :laughing:

Comment (nicklem) [reply docs/organizations/managing-people.md:18]: 😄 

```suggestion
To join or add an organization after completing the sign-up process, click **Organizations** on the top right-hand menu under your avatar:
```

File: docs/organizations/managing-people.md (hunk 1 above)
Comment (nicklem) [reply docs/organizations/managing-people.md:44]: Sounds good. I'd avoid the second "Codacy" at the end; it should be clear enough as-is.

```suggestion
1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails the newly added members inviting them to log in.
```

**Review by prcr — APPROVED** [review]
:shipit:

**Pull request merged**